pkg/operator/defaultscccontroller/defaultscccontroller.go

```diff
@@ -0,0 +1,211 @@
+package defaultscccontroller
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+
+	securityv1 "github.com/openshift/api/security/v1"
+	security1informers "github.com/openshift/client-go/security/informers/externalversions"
+	securityv1listers "github.com/openshift/client-go/security/listers/security/v1"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	ControllerName = "default-scc-upgradeable"
+)
+
+type Options struct {
+	Factory        security1informers.SharedInformerFactory
+	OperatorClient v1helpers.OperatorClient
+	Recorder       events.Recorder
+}
+
+func NewDefaultSCCController(options *Options) (controller *DefaultSCCController, err error) {
+	defaultSCCSet, err := NewDefaultSCCCache()
+	if err != nil {
+		err = fmt.Errorf("[%s] failed to render default SCC assets - %s", ControllerName, err.Error())
```

Comment:
nit: it's still a little odd style. Usually you see `return fmt.Errorf(...)` here. And usually you don't name the error arg at all.